Source/WebKit/Platform/IPC/SharedBufferReference.cpp

```diff
@@ -62,7 +62,7 @@ auto SharedBufferReference::serializableBuffer() const -> std::optional<Serializ
         return std::nullopt;
     if (!m_size)
         return SerializableBuffer { 0, std::nullopt };
-    auto sharedMemoryBuffer = m_memory ? m_memory : SharedMemory::copyBuffer(*m_buffer.copyRef());
+    auto sharedMemoryBuffer = m_memory ? m_memory : SharedMemory::copyBuffer(*RefPtr { m_buffer });
```

Comment:
cc @rniwa as this was a false positive.

Personally I would have used `Ref { *m_buffer }` as it is a bit shorter.

Reply:
Thanks for the suggestion, I'll do that.